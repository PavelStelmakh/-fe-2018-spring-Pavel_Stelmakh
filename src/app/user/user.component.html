<app-user-details  [role]="'user'"></app-user-details>
