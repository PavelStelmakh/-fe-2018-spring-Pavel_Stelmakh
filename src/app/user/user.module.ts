import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { Routes, RouterModule } from "@angular/router";
import { UserComponent } from './user.component';
import { UserDetailsModule } from '../user-details/user-details.module';
import { AuthGuard } from '../auth/auth.guard';

const route: Routes = [
  {
    path: '',
    component: UserComponent,
    canActivate: [AuthGuard]
  }
];

@NgModule({
  imports: [
    CommonModule,
    RouterModule.forChild(route),
    UserDetailsModule
  ],
  declarations: [UserComponent]
})
export class UserModule { }
