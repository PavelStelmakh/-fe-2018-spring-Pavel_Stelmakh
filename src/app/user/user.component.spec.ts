import { TestBed, async, ComponentFixture } from '@angular/core/testing';
import { NO_ERRORS_SCHEMA } from '@angular/core';
import { UserComponent } from './user.component';

describe('UserComponent', () => {
  let fixture: ComponentFixture<UserComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [
        UserComponent
      ],
      schemas: [
       NO_ERRORS_SCHEMA
      ]
    }).compileComponents();

  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(UserComponent);
  });

  it('renders markup to snapshot', () => {
    fixture.detectChanges();

    expect(fixture).toMatchSnapshot();
  });

});
