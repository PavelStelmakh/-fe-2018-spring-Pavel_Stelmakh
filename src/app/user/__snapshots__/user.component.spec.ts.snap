// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserComponent renders markup to snapshot 1`] = `
<app-user>
  <app-user-details />
</app-user>
`;
